### PR TITLE
IVE-186: gpio_interrupt

### PR DESCRIPTION
**Pull request by dmitryalexeevemcraft**

### CODE REVIEW

File: thingy9151lite_nrf5340_cpuapp_app/src/addons/gpio_shell.c [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+            pinstr = strtok(NULL, ".");
+            pin = strtol(pinstr, NULL, 10);
+            shell_print(sh, "P%u.%u: %u", block, pin, gpio_data->counters[pin]);
+        }
+
+        return 0;
+    }
+
+    for(block = 0; block <= 1; block++) {
+        for(pin = 0; pin < PIN_NUM_MAX; pin++ ) {
```
Comment (vladimirkhusainov-emcraft) [inline thingy9151lite_nrf5340_cpuapp_app/src/addons/gpio_shell.c:57]: Remove the space character after pin++.

File: thingy9151lite_nrf9151_app/src/addons/gpio_shell.c [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    int pin, block;
+    char *blockstr, *pinstr, *argvstr;
+
+    static const struct device *gpio0 = DEVICE_DT_GET_OR_NULL(DT_NODELABEL(gpio0));
+    static const struct device *gpio1 = DEVICE_DT_GET_OR_NULL(DT_NODELABEL(gpio1));
+    const struct device *gpio_dev;
+
+    struct gpio_nrfx_data *gpio_data = NULL;
+
+    if (argc > 1 ) {
```
Comment (vladimirkhusainov-emcraft) [inline thingy9151lite_nrf9151_app/src/addons/gpio_shell.c:39]: Remove the space character after "1"

File: thingy9151lite_nrf9151_app/src/addons/gpio_shell.c [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+        if (gpio_data != NULL) {
+            pinstr = strtok(NULL, ".");
+            pin = strtol(pinstr, NULL, 10);
+            shell_print(sh, "P%u.%u: %u", block, pin, gpio_data->counters[pin]);
+        }
+
+        return 0;
+    }
+
+    for(block = 0; block <= 1; block++) {
```
Comment (vladimirkhusainov-emcraft) [inline thingy9151lite_nrf9151_app/src/addons/gpio_shell.c:56]: Add a space after "for"

File: thingy9151lite_nrf9151_app/src/addons/gpio_shell.c [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+        for(pin = 0; pin < PIN_NUM_MAX; pin++ ) {
```
Comment (vladimirkhusainov-emcraft) [inline thingy9151lite_nrf9151_app/src/addons/gpio_shell.c:57]: Add a space after "for". 
Remove the space character after "pin++".

**Review by vladimirkhusainov-emcraft — APPROVED** [review]

**Review by VladimirKhusainov — APPROVED** [review]

**Pull request merged**